static/js/settings_org.js

```diff
@@ -259,18 +256,34 @@ function set_msg_edit_limit_dropdown() {
     settings_ui.disable_sub_setting_onchange(value !== "never", "id_realm_edit_topic_policy", true);
 }
 
+function message_delete_limit_setting_enabled() {
+    if (
+        page_params.realm_delete_own_message_policy ===
+        settings_config.common_message_policy_values.by_admins_only.code
+    ) {
+        return false;
+    }
+    if (
+        page_params.realm_delete_own_message_policy ===
+        settings_config.common_message_policy_values.by_moderators_only.code
+    ) {
+        return false;
+    }
+    return true;
+}
```

Comment:
Can you add a comment explaining the purpose of this function?

Reply:
Added the comment.